packages/documentation/src/stories/components/list/list.snapshot.stories.ts

```diff
@@ -0,0 +1,52 @@
+import type { Args, StoryContext, StoryObj, StoryFn } from '@storybook/web-components';
+import meta from './list.stories';
+import { html } from 'lit';
+import { bombArgs } from '@/utils';
+
+const { id, ...metaWithoutId } = meta;
+
+export default {
+  ...metaWithoutId,
+  title: 'Snapshots',
+};
+
+type Story = StoryObj;
+
+export const List: Story = {
```

Comment:
We will probably have snapshots for the HTML lists as well.
```suggestion
export const PostList: Story = {
```